fastestimator/trace/io/model_eval.py

```diff
@@ -0,0 +1,451 @@
+            package = Package('seqsplit')
+            if package not in tabular.packages:
+                tabular.packages.append(package)
+
+            # add table heading
+            tabular.add_row(("Total test #", "Total pass #", "Total fail #"), strict=False)
+            tabular.add_hline()
+
+            tabular.add_row((pass_num + fail_num, pass_num, fail_num), strict=False)
+
+    def _document_instance_table(self, tests: List[dict], with_ID: bool):
+        """Document a result table of per-instance tests.
+
+        Args:
+            tests: List of corresponding test dictionary to make a table.
+            with_ID: Whether the test information includes data ID.
+        """
+        if with_ID:
+            table_spec = '|c|X|c|c|X|'
+        else:
+            table_spec = '|c|X|c|c|'
+
+        with self.doc.create(Tabularx(table_spec, booktabs=True)) as tabular:
+            package = Package('seqsplit')
+            if package not in tabular.packages:
+                tabular.packages.append(package)
+
+            # add table heading
+            row_cells = [
+                MultiColumn(size=1, align='|c|', data="Test ID"),
+                MultiColumn(size=1, align='c|', data="Test description"),
+                MultiColumn(size=1, align='c|', data="Pass threshold"),
+                MultiColumn(size=1, align='c|', data="Failure count")
+            ]
+
+            if with_ID:
+                row_cells.append(MultiColumn(size=1, align='c|', data="Failure data instance ID"))
+
+            tabular.add_row(row_cells)
+            tabular.add_hline()
+
+            for idx, test in enumerate(tests, 1):
+                if idx > 1:
+                    tabular.add_hline()
+
+                des_data = [WrapText(data=x, seq_thld=27) for x in test["description"].split(" ")]
+                row_cells = [
+                    self.test_id,
+                    TabularCell(data=des_data, token=" "),
+                    NoEscape(r'$\le $' + str(test["fail_threshold"])),
+                    test["fail_number"]
+                ]
+                if with_ID:
+                    id_data = [WrapText(data=x, seq_thld=27) for x in test["fail_id"]]
+                    row_cells.append(TabularCell(data=id_data, token=", "))
+
+                tabular.add_row(row_cells)
+                self.test_id += 1
+
+    def _document_aggregate_table(self, tests: List[dict]) -> None:
+        """Document a result table of aggregate tests.
+
+        Args:
+            tests: List of corresponding test dictionary to make a table.
+        """
+        with self.doc.create(Tabularx('|c|X|X|', booktabs=True)) as tabular:
+            package = Package('seqsplit')
+            if package not in tabular.packages:
+                tabular.packages.append(package)
+
+            # add table heading
+            tabular.add_row((MultiColumn(size=1, align='|c|', data="Test ID"),
```

Comment:
same question about the MultiColumn

Reply:
this is for changing the column type only for this row.